src/test/cpp/component.idl

```diff
@@ -161,4 +164,19 @@ namespace Component
             static Windows.Foundation.Collections.IMap<String, Int32> Map();
         }
     }
+
+    namespace Composable
```

Comment:
This needs to preserve the generation of [overridable] interfaces and methods if we're validating parity with C++/WinRT.

Reply:
Agreed, I need to flesh out the testing. Going to do that in a separate branch and PR. I'm also looking into cmake code coverage in tandem to ensure we cover all our supported scenarios and code paths.